tests/CLI/modules/event_log_tests.py

```diff
@@ -0,0 +1,135 @@
+                        '"securityGroupId":"200",'
+                        '"securityGroupName":"test_SG"}'
+                    ),
+                    indent=4,
+                    sort_keys=True
+                )
+            },
+            {
+                'date': '2017-10-18T10:42:13.089536-05:00',
+                'event': 'Security Group Rule(s) Removed',
+                'label': 'test_SG',
+                'metadata': json.dumps(json.loads(
+                        '{"requestId":"2abda7ca97e5a1444cae0b9",'
+                        '"rules":[{"direction":"ingress",'
+                        '"ethertype":"IPv4",'
+                        '"portRangeMax":2001,"portRangeMin":2000,"protocol":"tcp",'
+                        '"remoteGroupId":null,"remoteIp":null,"ruleId":"800"}]}'
+                    ),
+                    indent=4,
+                    sort_keys=True
+                )
+            },
+            {
+                'date': '2017-10-18T10:42:11.679736-05:00',
+                'event': 'Network Component Removed from Security Group',
+                'label': 'test_SG',
+                'metadata': json.dumps(json.loads(
+                        '{"fullyQualifiedDomainName":"test.softlayer.com",'
+                        '"networkComponentId":"100",'
+                        '"networkInterfaceType":"public",'
+                        '"requestId":"6b9a87a9ab8ac9a22e87a00"}'
+                    ),
+                    indent=4,
+                    sort_keys=True
+                )
+            },
+            {
+                'date': '2017-10-18T10:41:49.802498-05:00',
+                'event': 'Security Group Rule(s) Added',
+                'label': 'test_SG',
+                'metadata': json.dumps(json.loads(
+                        '{"requestId":"0a293c1c3e59e4471da6495",'
+                        '"rules":[{"direction":"ingress",'
+                        '"ethertype":"IPv4",'
+                        '"portRangeMax":2001,"portRangeMin":2000,"protocol":"tcp",'
+                        '"remoteGroupId":null,"remoteIp":null,"ruleId":"800"}]}'
+                    ),
+                    indent=4,
+                    sort_keys=True
+                )
+            },
+            {
+                'date': '2017-10-18T10:41:42.176328-05:00',
+                'event': 'Network Component Added to Security Group',
+                'label': 'test_SG',
+                'metadata': json.dumps(json.loads(
+                        '{"fullyQualifiedDomainName":"test.softlayer.com",'
+                        '"networkComponentId":"100",'
+                        '"networkInterfaceType":"public",'
+                        '"requestId":"4709e02ad42c83f80345904"}'
+                    ),
+                    indent=4,
+                    sort_keys=True
+                )
+            }
+        ]
+
+        self.assertEqual(correctResponse, json.loads(result.output))
```

Comment:
In general, I'm used to a Python convention for test assertions args being 'actual' followed by 'expected', to help disambiguate in fiddly cases. (The mnemonic being 'a' comes before 'e'.) i.e:

    self.assertEqual(json.loads(result.output), correctResponse)

Is the convention the reverse in this repo?

Reply:
It flip flops. I'll go with Actual then Expected though. Makes sense.